lib/quadblockquiz_web/router.ex

```diff
@@ -42,7 +42,7 @@ defmodule QuadblockquizWeb.Router do
     get "/sbom", PageController, :redirect_to_well_known
     get "/.well-known/sbom", PageController, :sbom
 
-    live "/leaderboard/:board_id", LeaderboardLive.Show
+    # live "/leaderboard/:board_id", LeaderboardLive.Show
```

Comment:
@Japhetb why is this commented out?

Reply:
Also, this leaderboard is under which part? I am trying to look for the design on figma and I can't find it.